client_apis/python/src/cbapi/cbapi.py

```diff
@@ -339,17 +341,18 @@ def watchlist(self, id=None):
         r.raise_for_status()
         return r.json()
 
-    def watchlist_add(self, type, name, search_query, id=None, readonly=False, basic_query_validation=True):
+    def watchlist_add(self, type, name, search_query, id=None, readonly=False):
```

Comment:
Is there anything already using this method and passing a value for `basic_query_validation`?